docs/pages/desktop-access/active-directory.mdx

```diff
@@ -73,9 +80,11 @@ Click **NEXT**.
 The PowerShell script will output a Teleport configuration block. Copy this
 block to a temporary location. Click **Next**.
 
-On the Linux host you installed Teleport to run as the Desktop Access connector,
-edit `/etc/teleport.yaml` and paste the configuration provided by the output of
-the previous step. Review and edit the addition as necessarily for your environment:
+### Install the Teleport Desktop Service
+
+On the Linux host where you will run the Teleport Desktop Service, create a file
+called `/etc/teleport.yaml` and paste the configuration provided by the output
+of the previous step. The configuration file will resemble the following:
```

Comment:
Since the callout about a dedicated host is removed above should we, out of an abundance of caution, edit this to say:

```suggestion
On the Linux host where you will run the Teleport Desktop Service, create a file
called `/etc/teleport.yaml` and paste the configuration provided by the output
of the previous step. To add the service to an existing instance of Teleport, copy
just the `windows_desktop_service` block and add it to your existing file.

The configuration file will resemble the following:
```

Reply:
Works for me!